build/task-package.js

```diff
@@ -39,43 +40,115 @@ const IGNORE = [
   '^/NOTICE$',
 ];
 
-const packageApp = options => new Promise((resolve, reject) => {
-  packager(options, (err, packed) => {
-    if (err) {
-      reject(err);
-    } else {
-      if (packed.length !== 1) {
-        reject(new Error('Expected `electron-packager` to return only one path.'));
-      } else {
-        resolve(packed[0]);
-      }
-    }
-  });
-});
+const archIfNotx64 = ARCH === 'x64' ? '' : `-${ARCH}`;
+
+// electron-builder outputs its targets using different naming schemes depending
+// on architecture and platform. We want them to be fairly consistent so this
+// is a set of instructions to copy/zip files into the targets we want.
+const fileChanges = {
+  win32: {
+    copies: [
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}`,
+                 `tofino Setup ${getManifest().version}${archIfNotx64}.exe`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-win32-${ARCH}.exe`)],
+
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}`,
+                 `tofino-${getManifest().version}-full.nupkg`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${getManifest().version}-full.nupkg`)],
+
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}`, 'RELEASES'),
+       path.join(Const.PACKAGED_DIST_DIR, 'RELEASES')],
+    ],
+    zips: [
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}-unpacked`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+    ],
+  },
+  darwin: {
+    copies: [
+      [path.join(Const.PACKAGED_DIST_DIR, 'mac', `tofino-${getManifest().version}-mac.zip`),
```

Comment:
Yes, the "default" electron-builder target builds dmg and zip on OSX (the zip is needed for app updates).